meerk40t/kernel/settings.py

```diff
@@ -91,10 +92,37 @@ def read_persistent(
         @param default: default value if item does not exist.
         @return: value
         """
+        def listed(str_value):
+            dummy = str_value.split(";")
+            result = []
+            for item in dummy:
```

Comment:
Rather than `dummy` you could just do `for item in str_value.split(";")` and it would be more readable.